legend-depot-core-services/src/main/java/org/finos/legend/depot/services/api/entities/EntityClassifierService.java

```diff
@@ -18,9 +18,9 @@
 import org.finos.legend.depot.domain.entity.StoredEntity;
 import org.finos.legend.depot.domain.version.Scope;
 
-import java.util.Set;
+import java.util.List;
 
 public interface EntityClassifierService
 {
-    Set<StoredEntity> getEntitiesByClassifierPath(String classifierPath, String search, Scope scope, Integer limit);
+    List<StoredEntity> getEntitiesByClassifierPath(String classifierPath, String search, Scope scope, Integer limit);
```

Comment:
Why List ?

Reply:
Set is used in the previous flow because we want to make sure these entities are distinct, but they were actually super defensive and useless. They are even more useless after this rework because now we get the entities directly deserialized from Mongo